raiden-dapp/src/scss/dimensions.scss

```diff
@@ -1,3 +1,4 @@
 $first-child-padding: 80px;
 $list-header-padding-top: 30px;
 $dialog-button-height: 35px;
+$ios-safe-area: 44px;
```

Comment:
```suggestion
$ios-safe-area-height: 44px;
```

Reply:
This will actually be removed completely. We tested the dApp on an new iPhone yesterday and the safe area is not a problem when you run it as a progressive web app. @palango suggested to check if this can be adjusted in Capacitor somehow. But for now that's not an issue.

Reply:
Okay. So you will delete it in this PR? If so feel free to close the conversation when this happened. Else I would still rename it. It is code that gets merged, no matter how long we think **now** this will live.

Reply:
Yes, exactly, I will delete it.